web/src/app/services/data-store/data-store.service.ts

```diff
@@ -381,15 +397,20 @@ export class DataStoreService {
 
     const predefinedLois = this.db.collection(
       `${SURVEYS_COLLECTION_NAME}/${surveyId}/lois`,
-      ref => ref.where('predefined', 'in', [true, null])
+      ref =>
+        ref.where(FieldNumbers.LocationOfInterest.source, '==', Source.IMPORTED)
```

Comment:
Consider also updating the name of variable from `predefined` and `user` to `imported` and `fieldData` for consistency.